examples/sentiment/train_recursive_minibatch.py

```diff
@@ -240,6 +240,8 @@ def main():
         ['epoch', 'main/loss', 'validation/main/loss',
          'main/accuracy', 'validation/main/accuracy', 'elapsed_time']))
 
+    trainer.extend(extensions.ProgressBar(update_interval=10))
```

Comment:
Why `update_interval=10`?

Reply:
Do you mean why I'm not using the default value of 100? I thought that 100 was a bit too long, when testing locally. 1 would be fine too for this example I'd say. Do you have other suggestions?

Reply:
For the record, various examples are already using various intervals.

Reply:
>For the record, various examples are already using various intervals.

I'm aware of that. I was just curious how you decided to use 10 in this particular example.

>I thought that 100 was a bit too long, when testing locally. 1 would be fine too for this example I'd say.

So this answers my question.